Improves "Duplicate Queue Names" section (#165)
laravel/vapor-docs: 1.0/resources/queues.md

            - invoices
```

### Ensuring Unique Queue Names Across Projects and Environments

> When using the same custom queue names, you will need to ensure that the names are unique between projects and environments. One way to achieve this is to use the `SQS_SUFFIX` environment variable in each environment. For instance, you can add suffix values such as `-staging` and `-production` to the `SQS_SUFFIX` environment variable, and append it to the queue names specified in your vapor.yml file. A sample configuration is shown below:

Reviewer: ```suggestion
When using custom queue names, it is important to ensure the names are unique between projects and environments. One way to achieve this is to use the `SQS_SUFFIX` environment variable in each environment. For instance, you can add suffix values such as `-staging` and `-production` to the `SQS_SUFFIX` environment variable, and append it to the queue names specified in your vapor.yml file. A sample configuration is shown below:
```

:::danger Duplicate Queue Names

> When using custom queue names, it is important to ensure the names are unique between projects and environments. One easy way to accomplish this is by using the `SQS_SUFFIX` environment variable in each environment. This guarantees that custom queue names have a unique names in SQS.

Reviewer: ```suggestion
When using custom queue names, it is important to ensure the names are unique between projects and environments. One easy way to accomplish this is by using the `SQS_SUFFIX` environment variable in each environment. This guarantees that custom queues have a unique name in SQS.
```
